MonoGame.Framework/Platform/Graphics/GraphicsAdapter.Legacy.cs

```diff
@@ -59,11 +62,15 @@ internal GraphicsAdapter()
         }
 #endif
 
+        /// <inheritdoc />
```

Comment:
I may be wrong, but empty inheritdoc on Dispose might not create a docline. There also should be `cref=IDisposable.Dispose` with it.

Reply:
Only using `<inheritdoc />` correctly adds the documentation inheritance without needing to specify the `cref` attribute.

However, for clarity, if preferred to add the `cref` attribute, let me know
CC: @mrhelmut, @SimonDarksideJ

Reply:
So long as the interface it is inheriting from is documented, then Inheritdoc is the recommended way to go.